pkg/resource/target_group/sdk.go

```diff
@@ -483,7 +483,7 @@ func (rm *resourceManager) sdkUpdate(
 	}()
 	if delta.DifferentAt("Spec.Targets") {
 		added, removed := getTargetsDifference(latest.ko.Spec.Targets, desired.ko.Spec.Targets)
-		arn := (string)(*latest.ko.Status.ACKResourceMetadata.ARN)
+		arn := (string)(*desired.ko.Status.ACKResourceMetadata.ARN)
```

Comment:
@michaelhtm looks like the controller is panicking here, do you know why was/wasn't this included in previous patch?